### ISSUE TITLE
Cannot Start Test Instance in Meteor 1.3.3

### ISSUE THREAD

**Issue by joncursi** [practicalmeteor/mocha#48]
After upgrading from Meteor 1.3.2.4 to 1.3.3, my test suite no longer starts up due to the following error:

```
W20160614-13:57:19.036(-4)? (STDERR) /Users/jcursi/.meteor/packages/meteor-tool/.1.3.3.1qdjuco++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20160614-13:57:19.036(-4)? (STDERR)                            throw(ex);
W20160614-13:57:19.036(-4)? (STDERR)                                  ^
W20160614-13:57:20.218(-4)? (STDERR) Error: Can't find npm module 'has'. Did you forget to call 'Npm.depends' in package.js within the 'modules-runtime' package?
W20160614-13:57:20.219(-4)? (STDERR)     at Object.Npm.require (/private/var/folders/95/_xjkj28565n0drq52mspt9rc0000gp/T/meteor-test-runitqegl/.meteor/local/build/programs/server/boot.js:203:17)
W20160614-13:57:20.219(-4)? (STDERR)     at options.fallback (packages/modules-runtime/modules-runtime.js:21:1)
W20160614-13:57:20.220(-4)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:88:1)
W20160614-13:57:20.220(-4)? (STDERR)     at meteorInstall.node_modules.expect.lib.Expectation.js (node_modules/expect/lib/Expectation.js:11:1)
W20160614-13:57:20.220(-4)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160614-13:57:20.220(-4)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160614-13:57:20.220(-4)? (STDERR)     at meteorInstall.node_modules.expect.lib.index.js (node_modules/expect/lib/index.js:3:1)
W20160614-13:57:20.221(-4)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160614-13:57:20.221(-4)? (STDERR)     at Module.require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160614-13:57:20.221(-4)? (STDERR)     at Module.Mp.import (/Users/jcursi/.meteor/packages/modules/.0.6.2.1r5o2um++os+web.browser+web.cordova/npm/node_modules/reify/lib/runtime.js:52:16)
```

Per https://github.com/meteor/meteor/blob/master/History.md#v133 , it looks like `phantomjs` might be the culprit, however `meteor npm install -g phantomjs-prebuilt` did not solve the problem for me.

**Comment by jsep** [comment]
@joncursi , works for us:
1. What os and browser versions r u on?
2. Can you create a repo that reproduces the problem?

**Comment by jsep** [comment]
@joncursi I need more details in order to solve this issue, it's working great with that version of meteor including the latest version.

**Comment by joncursi** [comment]
@jsep it looks like this issue has been corrected in the combination of Meteor 1.3.4 + practicalmeteor:mocha 2.4.5_4 (not sure which had the bug). After running `meteor update` ...

```
updated to Meteor 1.3.4.             

Changes to your project's package version selections from updating package versions:

dispatch:mocha-core          removed from your project
dispatch:mocha-phantomjs     upgraded from 0.1.5 to 0.1.6
practicalmeteor:mocha        upgraded from 2.4.5_2 to 2.4.5_4
practicalmeteor:mocha-core*  upgraded from 0.1.4 to 1.0.0
```

the test suite is functional again for me.
